src/Advection/vector_invariant_advection.jl

```diff
@@ -213,6 +213,12 @@ end
     return + upwind_biased_product(û, ζᴸ, ζᴿ)
 end
 
+## Upwinding `δ` is not like upwinding `ζ`. `u` is trasporting `ζ`, while 
```

Comment:
It isn't helpful (for me) to mention upwinding for `ζ`. I don't know how `ζ` should be upwinded (or any quantity for that matter)...

Reply:
Why would we not reconstruct `A * ζ`? Have you tried that? It seems probable that any derivation of the scheme would have a similar implication for vorticity, right?